project/source/shared/Prefabs.cpp

```diff
@@ -219,226 +164,157 @@ Entity Create_(registry_type*, EntityInfo);
 template<> Entity Create_<Resource::Beeper>(registry_type* registry, EntityInfo info)
 {
     auto handle = registry->Add<Entity>(std::move(info));
-    registry->Add<Renderer>(handle, mesh::Beeper, material::Beeper);
+    registry->Add<vulkan::MeshRenderer>(handle, mesh::meshPaths[0], material::Beeper, nc::graphics::vulkan::TechniqueType::PhongAndUi);
```

Comment:
Indexing the vector like this seems brittle and confusing. I don't think we have to fix it right at this moment, but I'd like to find a better way.